### PR TITLE
Add link from Moment banner to pledge

### PR DESCRIPTION
**Pull request by tjsilver**
## What does this change?

This changes the URL of 'read our pledge' link on the Moment banner to go to the pledge page.

Tested locally.

## Screenshots

image

### CODE REVIEW

**Review by tomrf1 — APPROVED** [review]
👍

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-moment.js [hunk 1]
```diff
@@ -81,13 +81,11 @@ export const acquisitionsBannerMomentTemplate = (
 
             <div class="moment-banner__buttons">
                 <div class="engagement-banner__cta">
-                    <a tabindex="3" class="button  engagement-banner__button  engagement-banner__button--moment-link" href="${
-                        params.linkUrl
-                    }">
+                    <a tabindex="3" class="button  engagement-banner__button  engagement-banner__button--moment-link" href="https://www.theguardian.com/p/cfm63">
```
Comment (ripecosta) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-moment.js]: Is this URL supposed to exist already? I get a 404 when I try to access it.

Comment (tomrf1) [reply static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-moment.js]: it's the short url for an article that has not yet been published. The banner will not be switched on until it has

Comment (tjsilver) [reply static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-moment.js]: No, the link will go live when the Moment is launched.

Comment (ripecosta) [reply static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-moment.js]: 👍

**Pull request merged**